{{ cookiecutter.format }}/extras/{{ cookiecutter.installer_path }}/_run_post_install.bat

```diff
@@ -0,0 +1,14 @@
+@echo off
+setlocal enabledelayedexpansion
+SET ALLUSERS=%1
+{% for name in cookiecutter.install_options -%}
+{%- set argn = loop.index + 1 -%}
+SET {{ name.upper() }}=%{{ argn }}
+{% endfor -%}
+
+CALL "%~dp0post_install.bat"
```

Comment:
Why do the "run" scripts have leading underscores, but the actual post_install script doesn't? We're not using the user's original filename, so there's no chance of a clash.

Reply:
That's fair - my original thinking was picking a name to avoid collisions, but as you note, we're renaming the script, so that's a non-issue.